spec/workers/scheduler/user_cleanup_scheduler_spec.rb

```diff
@@ -9,6 +9,7 @@
   let!(:old_unconfirmed_user) { Fabricate(:user) }
   let!(:confirmed_user)       { Fabricate(:user) }
   let!(:moderation_note)      { Fabricate(:account_moderation_note, account: Fabricate(:account), target_account: old_unconfirmed_user.account) }
+  let!(:webauthn_credential)  { Fabricate(:webauthn_credential, user_id: old_unconfirmed_user.id) }
```

Comment:
As the linter already complained: I think you wanted to / should use this in an example to make sure it is deleted.